src/services/Api.ts

```diff
@@ -76,13 +77,17 @@ export const extendSession = () => {
 };
 
 export const login = async (
-  request: LoginRequest = { username: ANONYMOUS_USER, password: 'anonymous' }
+  request: LoginRequest = { username: ANONYMOUS_USER, password: 'anonymous', token: '' }
 ): Promise<Response<LoginSession>> => {
+  const params = new URLSearchParams();
```

Comment:
not sure if `const` conveys the right thing here. As the next line mutates the `params`.

Reply:
Typescript transpiler is not complaining 🤷‍♂️
I think `const` is only avoiding reassignment.

So, you choose. I can change to `let` if it looks better.

Reply:
+1 to changing it to `let`. Also, iirc `const` only guards against reassignment, you need to freeze the object to protect against mutations (which only happens at runtime :D).